eng/ci/templates/official/jobs/run-non-e2e-tests.yml

```diff
@@ -72,6 +81,7 @@ jobs:
       AzureWebJobsSecretStorageKeyVaultTenantId: $(AzureTenantId)
       AzureWebJobsSecretStorageKeyVaultClientId: $(AzureClientId)
       AzureWebJobsSecretStorageKeyVaultClientSecret: $(AzureClientSecret)
+      AzureWebJobsEventHubPath: testhub
```

Comment:
Should the value wrapped inside quotes? `"testhub"`?

Reply:
Not needed. Yaml will interpret both as a string.

Reply:
Nit: I feel having quotes help with readability that it is not a variable.